src/Spago/PackageSet.hs

```diff
@@ -231,31 +233,18 @@ rootPackagePath (Dhall.Import
 rootPackagePath _ = Nothing
 
 
--- | In a Monorepo we don't wish to rebuild our shared packages over and over, 
+-- | In a Monorepo we don't wish to rebuild our shared packages over and over,
 -- | so we build into an output folder where our root packages.dhall lives
 findRootOutputPath :: Spago m => System.IO.FilePath -> m (Maybe System.IO.FilePath)
 findRootOutputPath path = do
   echoDebug "Locating root path of packages.dhall"
   imports <- liftIO $ Dhall.readImports $ Text.pack path
   let localImports = mapMaybe rootPackagePath imports
-  pure $ (flip System.FilePath.replaceFileName) "output" <$> (findRootPath localImports)
+  pure $ flip System.FilePath.replaceFileName "output" <$> findRootPath localImports
 
 -- | Given a list of filepaths, find the one with the least folders
 findRootPath :: [System.IO.FilePath] -> Maybe System.IO.FilePath
-findRootPath paths 
```

Comment:
The only change not suggested by hlint, but I couldn't resist to simplify this to more readable one-liner..